mkdocs.yml

```
@@ -301,7 +301,7 @@ nav:
         - Encoders and Joint limits (manual): icub_testing/encoders_manual.md
       - Configure IP on a setup for ETH boards:
         - Configure IP on a setup for ETH boards: configure_static_ip/configure-static-ip.md
-      - Using iCub with ROS: icub_ros/index.md
+      - Using iCub/ergoCub with ROS2: icub-ergocub_ros2/index.md
```

```suggestion
      - Using iCub or ergoCub with ROS2: icub-ergocub_ros2/index.md
```

Nitpick, I think explicitly writing "or" is more readable.